podcasts/EpisodeBookmarksTabsView.swift

```diff
@@ -0,0 +1,50 @@
+import SwiftUI
+
+/// Displays a fake set of tabs that allows the user to open the bookmarks view from the podcast list
+struct EpisodeBookmarksTabsView: View {
+    @EnvironmentObject var theme: Theme
+
+    weak var delegate: PodcastActionsDelegate?
```

Comment:
I also believe we can use a `TabsDelegate` or something like that. But again, we can leave for later.